index.js

```diff
@@ -26,8 +27,10 @@ function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
+  // this disables the submit button if it's greater than 99  
+  // end of added code
 
-  hideAllMessages();
+//  hideAllMessages();
```

Comment:
Once the fix in `hideAllMessages` is done (see my comment below), we should uncomment this line so the function can run after every guess input, otherwise the messages stack up.